src/App/App.js

```diff
@@ -10,6 +10,19 @@ import useSearchResults from '../hooks/useSearchResults.js';
 
 function App() {
   const { isLoading } = useAuth0();
+  
+  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russel Brand', 'Bernie Madoff', 'Amy Winehouse']
+  const { initialResults, controversies, triggerSearch  } = useSearchResults();
```

Comment:
thank you for fixing it! defintiely a saving grace to have that oversight on the project